lib/reporters/tap.js

```diff
@@ -118,6 +119,21 @@ class TapReporter {
 		}) + os.EOL);
 	}
 
+	writeLogs(evt, flags = {error: false, title: ''}) {
+		let formattedTitle = flags.title || this.prefixTitle(evt.testFile, evt.title);
```

Comment:
This would be better as `writeLogs(evt, { error = false, title = this.prefixTitle(evt.testFile, evt.title) } = {})`.

Reply:
I will use: `writeLogs(evt, { error = false, title = this.prefixTitle(evt.testFile, evt.title) }) {` without default value to prevent bad behavior, in that case if you don;t pass the object deconstruction fails and you are acknowledged